tutorcodejail/templates/codejail/build/codejail/Dockerfile

```diff
@@ -51,9 +51,8 @@ WORKDIR /var/tmp
 RUN mkdir -p common/lib/
 
 COPY --from={{ DOCKER_IMAGE_OPENEDX }} /openedx/edx-platform/common/lib/sandbox-packages common/lib/sandbox-packages
-COPY --from={{ DOCKER_IMAGE_OPENEDX }} /openedx/edx-platform/common/lib/symmath common/lib/symmath
```

Comment:
Do we not need symmath anymore? or is it included in py38?

Reply:
I see.

Now that it is located at openedx-calc
> Symmath has been added to openedx-calc alongside calc in https://github.com/openedx/openedx-calc/pull/38.

Do we need to allow it or install it or something else?

Reply:
Thanks! I'll add it as a dependency so we install the same requirements as before